CMakeModules/osx_install/OSXInstaller.cmake

```diff
@@ -261,10 +296,10 @@ IF(BUILD_GRAPHICS)
     PKG_BUILD(  PKG_NAME        ForgeLibrary
                 DEPENDS         OSX_INSTALL_SETUP_FORGE_LIB
                 TARGETS         forge_lib_package
-                INSTALL_LOCATION /usr/local/
+                INSTALL_LOCATION /usr/local/lib
```

Comment:
This doesn't look right. Was it really installing the libraries in the /usr/local directory? I don't see a change in the PKG_BUILD function that is associated with this change.